fetcher/src/job/job.controller.ts

```diff
@@ -13,8 +18,34 @@ export class JobController {
 
   constructor(@InjectQueue(FETCHER_QUEUE_NAME) private queue: Queue) {}
 
+  async onModuleInit() {
+    const chain = process.env.CHAIN
+    const aggregators = await loadActiveAggregators({ chain, logger: this.logger })
+
+    for (let aggregator of aggregators) {
+      this.startFetcher({ aggregatorHash: aggregator.aggregatorHash, chain, isInitial: true })
```

Comment:
`startFetcher` is asynchronous function. I think you should call it with `await`.